### PR TITLE
Install cleanup

### PR DESCRIPTION
**Pull request by MBishop17**
Need a review on install doc cleanup.

### CODE REVIEW

File: content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md [hunk 1]
```diff
@@ -2,67 +2,69 @@
...
-	After you've completed configuration of the config file template, back the config file up in a safe place. You can reuse this file for upgrades later.
+	After you've completed configuration of the RKE config file: 1. it's no longer a template, since you'll be using it, and 2. you should backup the RKE config file to a safe place. You can reuse this file for upgrades later.
 
-10. [Run RKE](#part-10-run-rke)
+10. [Run RKE](#10-run-rke)
 
 	Run RKE to deploy Rancher to your cluster.
 
-11. [Backup Config File](#part-11-backup-config-file)
+11. [Backup Auto-Generated Config File](#11-backup-auto-generated-config-file)
```
[the remark this answers is not in the document]
Comment (MBishop17) [reply content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md]: They're two separate files. One is an RKE config file that you create, and the other one is a RKE config that's automatically generated by `RKE up`. Let me double check with Bill that both files are needed. I can't think of a reason you'd need the former.

File: content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md [hunk 2]
```diff
@@ -99,222 +101,246 @@ We have example configurations for the following load balancers:
...
-## Part 9-Backup Your YAML File
+## 9. Backup Your RKE Config File
 
-After you close your `.yml` file, back it up to a secure location. You can use this file again when it's time to upgrade Rancher.
+After you close your RKE config file, `rancher-cluster.yml`, back it up to a secure location. You can use this file again when it's time to upgrade Rancher.
 
-## Part 10-Run RKE
+## 10. Run RKE
 
 All configuration is in place to run RKE. You can do this by running the `rke up` command and using the `--config` parameter to point to your config file.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md]: This first sentence sounds weird to me. Maybe something like `With all the configuration in place, we can use RKE to launch Rancher.`

File: content/rancher/v2.x/en/installation/ha-server-install/_index.md [hunk 3]
```diff
@@ -1,68 +1,72 @@
 ---
 title: High Availability Installation
 weight: 275
+aliases:
+  - /rancher/v2.x/en/installation/after-installation/etcd-backup-and-restoration/
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/ha-server-install/_index.md]: why is this added here?

File: content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md [hunk 4]
```diff
@@ -99,222 +101,246 @@ We have example configurations for the following load balancers:
...
-## Part 5-Download Config File Template
+3.  Confirm that RKE is now executable by running the following command:
 
-RKE uses a `.yml` config file to install and configure your Kubernetes cluster. There are 2 templates to choose from, depending on the SSL certificate you want to use.
+    ```
+    # MacOS
+    $ ./rke_darwin-amd64 -version
+    # Linux
+    $ ./rke_linux-amd64 -version
+    ```
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/ha-server-install-external-lb/_index.md]: --version instead of -version

File: content/rancher/v2.x/en/installation/ha-server-install/_index.md [hunk 5]
```diff
@@ -133,154 +152,168 @@ docker run -d --restart=unless-stopped \
...
 
-## Part 5-Download Config File Template
+3.  Confirm that RKE is now executable by running the following command:
+
+    ```
+    # MacOS
+    $ ./rke_darwin-amd64 -version
+    # Linux
+    $ ./rke_linux-amd64 -version
+    ```
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/ha-server-install/_index.md]: --version instead of -version
